h2o-bindings/bin/custom/R/gen_word2vec.py

```diff
@@ -22,12 +22,12 @@
 
 # Import the CraigslistJobTitles dataset
 f <- "https://raw.githubusercontent.com/h2oai/sparkling-water/rel-1.6/examples/smalldata/"
-jobtitles <- h2o.importFile(paste0(f, "craigslistJobTitles.csv"), 
-                             col.names = c("category", "jobtitle"),
+job_titles <- h2o.importFile(paste0(f, "craigslistJobTitles.csv"), 
+                             col.names = c("category", "job_title"),
```

Comment:
>col.names = c("category", "job_title")

This change is not correct and should be reverted. So what's happening is, the craigslistJobTitles.csv dataset includes two columns with headers: one named category and one named jobtitle. Your change here causes a new header row to be created and pushes the original header row into the data (see below).

```
> job_titles
   category                                            job_title
1  category                                             jobtitle
2 education                              After School Supervisor
3 education *****TUTORS NEEDED - FOR ALL SUBJECTS, ALL AGES***** 
```